help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -99,12 +100,13 @@ Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]** search for [!DNL
 ### Authenticate to destination {#authenticate}
 
 To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
-![Platform UI screenshot showing how to authenticate.](../../assets/catalog/crm/microsoft-dynamics-365/authenticate-destination.png)
+![Platform UI screenshot showing how to authenticate.](../../assets/catalog/crm/microsoft-dynamics-365/authenticate-destination1.png)
```

Comment:
let's not add clutter. instead of adding a new screenshot, let's just update the old `microsoft-dynamics-365/authenticate-destination.png` screenshot to show the new **Region** dropdown selector.